ivadomed/scripts/download_data.py

```diff
@@ -283,7 +275,7 @@ def main(args=None):
     data_name = arguments.d
 
     if arguments.output is None:
-        dest_folder = os.path.join(os.path.abspath(os.curdir), data_name)
+        dest_folder = str(Path(Path.cwd().absolute(), data_name))
```

Comment:
Apparently empty Path() is just cwd. But I think better to be more explicit here.